MAINTAINERS.md

```diff
@@ -1,10 +1,10 @@
-# Maintainers
+# Project Maintainers
```

Comment:
Should we put committers in MAINTAINERS.md?

Reply:
I think it is ok. @dcalvin PTAL

Reply:
I think it's good as these two are different roles with definitions.